packages/uikit-workshop/webpack.config.js

```diff
@@ -1,10 +1,8 @@
 // webpack.config.js
-const CleanWebpackPlugin = require('clean-webpack-plugin');
-const HardSourceWebpackPlugin = require('hard-source-webpack-plugin-patch');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const NoEmitPlugin = require('no-emit-webpack-plugin');
 const autoprefixer = require('autoprefixer');
-const CriticalCssPlugin = require('critical-css-webpack-plugin');
```

Comment:
@sghoweri: this seems to have been removed (intentionally?) by https://github.com/pattern-lab/patternlab-node/commit/2cb663d053e004ab17eb5af606ce289834bb89fc#diff-8c9b7dba764c6bd34aaec36a6a42e7b69d9a46b19712a47706355bb8616d3319L365 - is that correct ? Then I'd like to remove the module now

Reply:
Please proceed by removing it 👍